app/channels/queue_channel.rb

```diff
@@ -50,6 +50,16 @@ def update_instructor_message(data)
 
   end
 
+  def update_ping_message(data)
```

Comment:
Did you have a use case in mind for why the ping message would be shared across instructors?

Reply:
You're right, it doesn't make sense to structure the code this way so I've changed it. Let me know if this works!